environments/sbox/sbox.tfvars

```diff
@@ -0,0 +1,7 @@
+env = "sbox"
+product = "neuvector"
+location = "uksouth"
+key_vault = {
+  name = "dtssharedservicessboxkv"
```

Comment:
we shouldn't be adding vars per environment they can be determined from a pattern if we can avoid it, this uses the env flag in it

Reply:
agree, this was just for testing purposes